1_task_1-15.py

```diff
@@ -0,0 +1,308 @@
+
+
+# 8. Write a Python program to remove blank elements from a given array.
+# Sample Output:
+# Original array:
+# ["Red", "Green", "", "Blue", "White"]
+# Remove a blank element from the above array:
+# ["Red", "Green", "Blue", "White"]
+
+original = ["Red", "Green", "", "Blue", "White"]
+
+
+def remove_blank(array):
+    without_blank = []
+    for element in array:
+        if element:
+            without_blank.append(element)
+    print(without_blank)
+
+
+remove_blank(original)
+
+# or
+# print([element for element in original if element != ""])
+# or
+# print([element for element in original if element.strip()])
+
+
+# 9. Write a Python program to split a delimited string into an array.
+# Sample Output:
+# Original delimited string:
+# Red, Green, Blue, White1, 3, 4, 5, 7String to array:
+# ["Red", " Green", " Blue", " White"]
+# [1, 3, 4, 5, 7]
+import re
+
+delimited_string = 'Red, Green, Blue, White1, 3, 4, 5, 7'
+
+
+def string_to_array(string):
+    template_words = '[A-Z][a-z]{2,}'
+    template_digit = r'\d'
+
+    words = re.findall(template_words, string)
+    digits = re.findall(template_digit, string)
+
+    print(words)
+    print(list(map(int, digits)))
+# or
+    # separable_index = string.find('1')
+    # words = string[:separable_index].split(',')
+    # digits = string[separable_index:].split(',')
+    # print(words)
+    # print([int(x) for x in digits])
+
+
+string_to_array(delimited_string)
+
+
+# 10. Write a Python program to create an array with the elements "rotated left"
+# of a given array of ints length 3.
+# Sample Output:
+# [2, 5, 1]
+# [2, 3, 1]
+# [2, 4, 1]
+
+given_array = [1, 2, 5]
+
+
+def rotated_left(array):
+    rotated_list = [array[1], array[2], array[0]]
+    print(rotated_list)
+
+
+rotated_left(given_array)
+
+
+# 11. Write a Python program to create a new array with the elements in reverse order
+# from a given an array of integers length 3.
+
+given_array = [1, 2, 3]
+
+
+def reverse_order(array):
+    print(array[::-1])
+
+
+reverse_order(given_array)
+
+
+# 12. Write a Python program to find the larger between the first
+# and last elements of a given array of integers and length 3.
+# Replace all the other values to be that value. Return the changed array.
+
+given_array = [1, 8, 3]
+
+
+def larger_and_replace(array):
+    print(array)
+
+    max_element = max(array[0], array[-1])
+    print([max_element]*3)
+# or
+    # if array[0] > array[-1]:
+    #     print([array[0]]*3)
+    # else:
+    #     print([array[-1]]*3)
+
+
+larger_and_replace(given_array)
+
+
+# 13. Write a Python program to concatenate an array of arrays into one.
+
+given_array = [[1, 2, 3], [4, 5], [6, 7, 8, 9]]
+
+
+def concatenate(array):
+    concatenate_list = []
+    for i in array:
+        for x in i:
+            concatenate_list.append(x)
+    print(concatenate_list)
+
+
+concatenate(given_array)
+
+
+# 14. Write a Python program to check if a given array of integers contains 3 twice, or 5 twice.
```

Comment:
The script should return True for [2,3,4,5,3] or [4,3,6,7,5,7,8,5,9]. For [6,5,7,7] it should return False.